### PR TITLE
PHP: change phpunit version from 4.8.36 to 5.7.27

### PR DESCRIPTION
**Pull request by HannahShiSFB**
Update PHPUnit version from 4.8.36 to 5.7.27

### CODE REVIEW

File: tools/internal_ci/helper_scripts/prepare_build_macos_rc [hunk 1]
```diff
@@ -79,5 +79,3 @@ rm -rf ~/Library/Developer/Xcode/DerivedData
 mkdir -p ~/Library/Developer/Xcode
 ln -s /tmpfs/DerivedData ~/Library/Developer/Xcode/DerivedData
 
-# PHP tests currently require using an older version of PHPUnit
```
Comment (stanley-cheung) [inline tools/internal_ci/helper_scripts/prepare_build_macos_rc]: We might need to keep these 2 lines

**Comment by stanley-cheung** [comment]
There are mismatches - so please run `./tools/buildgen/generate_projects.sh`. There are diffs on these 2 files
```
  modified:   tools/dockerfile/interoptest/grpc_interop_php/Dockerfile                                        
  modified:   tools/dockerfile/test/php_jessie_x64/Dockerfile 
```

File: tools/internal_ci/helper_scripts/prepare_build_macos_rc (hunk 1 above)
Comment (stanley-cheung) [reply tools/internal_ci/helper_scripts/prepare_build_macos_rc]: Yea I think we need to keep these lines

File: templates/tools/dockerfile/php_deps.include [hunk 2]
```diff
@@ -4,4 +4,8 @@
 # Install dependencies
 
 RUN apt-get update && apt-get install -y ${'\\'}
-    git php5 php5-dev phpunit unzip
+    git php5 php5-dev unzip
+
+RUN wget https://phar.phpunit.de/phpunit-5.7.27.phar && \
```
Comment (stanley-cheung) [inline templates/tools/dockerfile/php_deps.include]: If you replace `\` with `${'\\'}`, the linebreaks will work better

File: tools/internal_ci/helper_scripts/prepare_build_macos_rc (hunk 1 above)
Comment (HannahShiSFB) [reply tools/internal_ci/helper_scripts/prepare_build_macos_rc]: reverted

File: templates/tools/dockerfile/php_deps.include (hunk 2 above)
Comment (HannahShiSFB) [reply templates/tools/dockerfile/php_deps.include]: modified

File: tools/dockerfile/distribtest/php_jessie_x64/Dockerfile [hunk 3]
```diff
@@ -14,4 +14,8 @@
 
 FROM debian:jessie
 
-RUN apt-get update && apt-get install -y php5 php5-dev php-pear phpunit
+RUN apt-get update && apt-get install -y php5 php5-dev php-pear
+
+RUN wget https://phar.phpunit.de/phpunit-5.7.27.phar && \
```
Comment (stanley-cheung) [inline tools/dockerfile/distribtest/php_jessie_x64/Dockerfile]: There is a problem with this specific file: `wget` was not previously installed.

**Pull request merged**